lib/renaissance_web/templates/register/register.html.eex

```diff
@@ -0,0 +1,15 @@
+<h1>Register</h1>
+<hr>
+<form action="/register" method="post">
```

Comment:
Use the [form helpers](https://hexdocs.pm/phoenix_html/Phoenix.HTML.Form.html#module-with-changeset-data). This will do CSRF for you, https://hexdocs.pm/phoenix_html/Phoenix.HTML.Form.html#module-csrf-protection.

It's great that you kept CSRF in mind, and figured out how to do this yourself though.